force-app/main/default/classes/SOQL.cls

```diff
@@ -314,18 +341,29 @@ public with sharing class SOQL {
      */
     public static void cloneLineItems(Id origOppId, List<Id> cloneOppIds) {
         // Query the original Opportunity Line Items
-
+        List<OpportunityLineItem> oppItems = [SELECT Quantity, OpportunityId, PricebookEntryId, UnitPrice
+                                            FROM OpportunityLineItem 
+                                            WHERE OpportunityId = :origOppId];
         // Create a list to hold the new line items
-
+        List<OpportunityLineItem> clonedOppItems = new List<OpportunityLineItem>();
         // Generate a random number between 1 and 3
-
+        Integer randomNumber = Integer.valueof(Math.random()*10);
+        if(randomNumber > 3){
+            randomNumber = 3;
+        }
```

Comment:
This is not the best way to generate this number. How can you make sure the that number will be between and 1 and 3?